### PR TITLE
POWR-1558a ESLint and StyleLint

### PR DESCRIPTION
**Pull request by joekarasek**

### CODE REVIEW

**Comment by joshuami** [comment]
Looks like there is a merge conflict with package.json that needs to be resolved as well.

**Comment by joshuami** [comment]
@joekarasek, please rebase or merge in master and clean up the conflict on this one. I'll try and get this PR merged today.

File: web/themes/custom/cloudy/package.json [hunk 1]
```diff
@@ -12,10 +12,16 @@
     "clean:pl": "rm -rf ../../../pattern-lab",
     "clean:wp": "rm -rf dist",
     "clean:migration1557": "rm -rf css js dist-cloudy dist-pl source",
+    "lint": "npm-run-all --parallel --continue-on-error --print-label lint:js lint:css",
+    "lint:css": "stylelint ./**/*.scss",
+    "lint:css:fix": "stylelint ./**/*.scss --fix",
+    "lint:fix": "npm-run-all --sequential --continue-on-error --print-label lint:js:fix lint:css:fix",
+    "lint:js": "eslint . --config ./.eslintrc.json",
+    "lint:js:fix": "eslint . --config ./.eslintrc.json --fix",
```
Comment (EvanLovely) [inline web/themes/custom/cloudy/package.json]: The use of the paths in `eslint .` and `stylelint **/*.scss` will result in slower commands due to them going into `node_modules` to build the list even though they then exclude them afterwards. Better to point them to sub directories first.

**Comment by joekarasek** [comment]
🤔 Seems like there was a build hiccup around the db and drupal's config files (behat was failing on configuration changes detected with entity types). Kicked off two fresh builds and it looks like the are now passing.

**Comment by joshuami** [comment]
This has some merge conflicts after merging in POWR-1782 and POWR-1783. Please pull in master and push that up for this to get added.

**Pull request merged**